src/runtime/server.ts

```diff
@@ -178,6 +178,17 @@ export async function createServer(
           if (routeInfo.access === 'private') {
             res.status(403).send('This asset has been marked as private');
           } else {
+            if (req.method === 'OPTIONS') {
+              res.set({
+                'access-control-allow-origin': '*',
+                'access-control-allow-headers':
+                  'Accept, Authorization, Content-Type, If-Match, If-Modified-Since, If-None-Match, If-Unmodified-Since',
+                'access-control-allow-methods': 'GET, POST, OPTIONS',
+                'access-control-expose-headers': 'ETag',
+                'access-control-max-age': '86400',
+                'access-control-allow-credentials': true,
+              });
```

Comment:
Does the options request actually return the content? if not we should probably return the same payload here.

Reply:
I don't believe an `OPTIONS` request to a Twilio Asset returns content, no. 

Do you mean this should avoid returning the content then? That is, it should be more like:

```javascript
        if (routeInfo.filePath) {
          if (routeInfo.access === 'private') {
            res.status(403).send('This asset has been marked as private');
          } else if (req.method === 'OPTIONS') {
            res.set({
              'access-control-allow-origin': '*',
              'access-control-allow-headers': 'Accept, Authorization, Content-Type, If-Match, If-Modified-Since, If-None-Match, If-Unmodified-Since',
              'access-control-allow-methods': 'GET, POST, OPTIONS',
              'access-control-expose-headers': 'ETag',
              'access-control-max-age': '86400',
              'access-control-allow-credentials': true,
            });
            res.status(200).end();
          } else {
            res.sendFile(routeInfo.filePath);
          }
        } else {
          res.status(404).send('Could not find asset');
        }
```

That is, send an empty body to an options request and send the file if it's not options.

Reply:
yeah :)